src/libs/ReportUtils.ts

```diff
@@ -428,6 +428,7 @@ type OptionData = {
     hasDraftComment?: boolean | null;
     keyForList?: string;
     searchText?: string;
+    private_isArchived?: string | null;
     isIOUReportOwner?: boolean | null;
     isArchivedRoom?: boolean | null;
```

Comment:
@srikarparsi is it fine to keep both (private_isArchived, isArchivedRoom) in OptionData?

Reply:
Sorry, didn't see this. Please correct me if I'm wrong but OptionData includes all the attributes from Report right (which already has private_isArchived)? So do we need to declare this twice?